modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -317,10 +325,124 @@ Status description:: error: transaction rollback - constituent rollback failed.
+
+Status description:: error: syntax error or access rule violation - number of node labels below supported minimum
+
+=== 42010
+
+Status description:: error: syntax error or access rule violation - number of node labels exceeds supported maximum
+
+=== 42011
+
+Status description:: error: syntax error or access rule violation - number of node properties exceeds supported maximum
+
+=== 42012
+
+Status description:: error: syntax error or access rule violation - number of node type key labels below supported minimum
+
+=== 42013
+
+Status description:: error: syntax error or access rule violation - number of node type key labels exceeds supported maximum
+
+=== 42014
+
+Status description:: error: syntax error or access rule violation - number of edge type key labels below supported minimum
+
+=== 42015
+
+Status description:: error: syntax error or access rule violation - number of edge type key labels exceeds supported maximum
+
+=== 42I00
+
+Status description:: error: syntax error or access rule violation - invalid case expression. 'CASE' expressions must have the same number of 'WHEN' and 'THEN' operands.
+
+=== 42I01
+
+Status description:: error: syntax error or access rule violation - invalid FOREACH. Invalid use of `{ $clause }` inside 'FOREACH'.
+
+=== 42I02
+
+Status description:: error: syntax error or access rule violation - invalid comment. Failed to parse comment. A comment starting with `'/*'` must also have a closing `'*/'`.
+
+=== 42I06
+
+Status description:: error: syntax error or access rule violation - invalid input. Invalid input `{ $input1 }`, expected: `{ $input2 }`.
+
+=== 42I18
+
+Status description:: error: syntax error or access rule violation - invalid reference to implicitly grouped expressions. The aggregation column contains implicit grouping expressions referenced by the variables `{ $varList }`. Implicit grouping expressions are variables not explicitly declared as grouping keys.
+
+
+=== 42I20
+
+Status description:: error: syntax error or access rule violation - invalid symbol in expression. Label expressions and relationship type expressions cannot contain `{ $input }`. To express a label disjunction use `{ $labelExpr }` instead.
+
+=== 42I25
+
+Status description:: error: syntax error or access rule violation - invalid use of CALL IN TRANSACTIONS. 'CALL { ... } IN TRANSACTIONS' is not supported after a write clause.
+
+=== 42I29
+
+Status description:: error: syntax error or access rule violation - invalid use of IS. The IS keyword cannot be used together with multiple labels in `{ $input }`. Rewrite the expression as `{ $replacement }`.
+
+=== 42I31
+
+Status description:: error: syntax error or access rule violation - invalid use of MATCH. MATCH ...' cannot directly follow an 'OPTIONAL MATCH ...'. Use a WITH clause between them.
+
+
+=== 42I37
+
+Status description:: error: syntax error or access rule violation - invalid use of `RETURN *`. `RETURN *'` is not allowed when there are no variables in scope.
```

Comment:
```suggestion
Status description:: error: syntax error or access rule violation - invalid use of `RETURN *`. `'RETURN *'` is not allowed when there are no variables in scope.
```

Same starting on single quote Sheet bug as above. Then I am not sure if we should double quote it in docs or not